test/plugins/plugin_test.rb

```diff
@@ -129,4 +129,11 @@ class TestPluginWithoutCustomValidators < ::Proxy::Plugin; end
   def test_custom_validators_returns_empty_hash_if_mappings_were_omitted
     assert TestPluginWithoutCustomValidators.custom_validators.empty?
   end
+
+  class TestCapabilityPlugin < ::Proxy::Plugin
```

Comment:
Can you add a cabability test which uses a lambda as well?

I'm getting an error once I restart my proxy when defining a capability like this:
```
capability(lambda{ ['a', 'b'] })
```

```
Error during startup, terminating. comparison of String with Proc failed
```

I think your tests in root_v2_api_tests didn't reveal it because of the way you're stubbing the `loaded` method but I'm just guessing

Reply:
Won't hurt.

Reply:
this was added